blocks/section-hero/section-hero.js

```diff
@@ -0,0 +1,140 @@
+import {
+  createOptimizedPicture, decorateIcons, getMetadata, readBlockConfig, toClassName,
+} from '../../scripts/lib-franklin.js';
+
+export default function decorate(block) {
+  const data = readBlockConfig(block);
+  const link = block.querySelector('a');
+  block.innerText = '';
+
+  // change the background color of the block based on the url
```

Comment:
Move these style changes to the CSS.

Refer to how we did this in the Card block.
https://github.com/hlxsites/24life/blob/add0940938a86c653b8fdc1a64a5f37eaf149971/blocks/card/card.js#L58